source/agora/common/crypto/ECC.d

```diff
@@ -268,6 +268,12 @@ public struct Point
         return result;
     }
 
+    /// Convenience overload to allow this to be converted to a `PublicKey`
+    public inout(ubyte)[] opSlice ( ) inout nothrow @nogc
```

Comment:
Why `inout` though? I think this should be `const`. Is there a use-case for modifying the internal data?

Reply:
If I apply the `const` to the function, I get an error like this.
`source/agora/common/Hash.d(133,21): Error: mutable method agora.common.crypto.ECC.Point.opSlice is not callable using a const object`

Reply:
`const` should be on both sides. Try that.

Reply:
It works. Thanks.